test/test_atomic_opener.py

```diff
@@ -0,0 +1,145 @@
+from pathlib import Path
+from rohmu.atomic_opener import atomic_opener
+
+import errno
+import os
+import pytest
+import time
+
+
+def _verify_file_not_created_and_dir_not_polluted(output_file: Path):
```

Comment:
return type hints are missing on all functions

Reply:
I can fix that.

Maybe I'm misremembering, though, I need a mypy cheatsheet: if there's at least one typed argument, mypy should perform type checking nevertheless, and treat return type as None by default, isn't this the default behaviour?

Reply:
mypy will usually type check (depending on its settings, which we need to recheck on this repo).

However the default type, if untyped, is not `None`, it's `Any`. So if you have something like:

```python
def check_is_odd(x: int):
    return x % 2

def test_something():
   x = something_give_me_a_value()
   check_is_odd(x)
```

The caller and the callee did not agree on whether it's supposed to be a boolean function or an assert-like function,
By not typing the return type, you miss an opportunity to be warned about it.

(This is a simplified but seen-before bug :) )

Reply:
But we really need to fix the mypy settings, so that everybody writes the full function prototype before review-time :)